cdc-server/README.md

```diff
@@ -1,97 +1,39 @@
-# Debezium Server
```

Comment:
Instead of removing these sections, lets keep it and add a new section for release.